.claude/skills.example/README.md

```
@@ -0,0 +1,24 @@
+
+This directory serves as an example for creating your own local Claude Code skills.
+
+## Usage
+
+1. Copy this directory to `.claude/skills/`
+2. Modify or add your own custom skills
+3. Your local skills will be automatically ignored by git
+
+## Example Skills Structure
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)